code/src/java/pcgen/gui2/csheet/CharacterSheetPanel.java

```diff
@@ -196,15 +174,13 @@ public void run()
 						}
 						catch (Throwable e)
 						{
-							final String errorMsg = "<html><body>Unable to process sheet<br>" +
-									e + "</body></html>";
-							ByteArrayInputStream instream = new ByteArrayInputStream(errorMsg.getBytes());
-							try
-							{
+							final String errorMsg = String.format("<html><body>Unable to process sheet<br>%s</body></html>", e);
```

Comment:
I didn't count the exact length, but since we are changing code in this file, we should try to wrap at least the new lines to a good line length.  Our standard is 80, but our warnings are currently higher.  I think 90 or 100 is fine, but shouldn't be too long.